code/style.css

```diff
@@ -1,4 +1,366 @@
+}
+
+.winner {
+  width: 100%;
+  max-height: auto;
+}
+
+/*** Footer styling ***/
+
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 100%;
+}
+
+.linkedin {
+  width: 30px;
+}
+
+/*** Media queries for tablet screens - min width 668px ***/
+
+@media (min-width: 668px){
+  body{
+    margin: 50px;
+  }
+  
+  .cards { 
+  grid-template-columns: repeat(2, 2fr);
+  }
+
+  .card {
+    height: 25rem;
+    grid-column: span 2;
+  }
+
+  .banner {
+    max-width: 300px;
+  }
+
+  .header { 
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-evenly;
+  }
+
+  .nav {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-items: space-evenly;
+  }
+
+  .flex-parent {
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: center;
+    align-items: center;
+  }
+
+   .one { 
+    width: 60%;
+  }
+  
+  .two { 
+    width: 40%;
+  }
+  
+  video {
+    height: 250px;
+  }
+
+  .cards {
+    display: grid; 
+    grid-gap: 20px;
+  }
+
+  .card {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+  }
+
+  .logo {
+    width: 200px;
+  }
+
+  .pauline {
+    width: 200px;
+  }
+
+  .course {
+    width: 200px;
+  }
+
+  .winner {
+    width: 200px;
+  }
+
+/*** Media queries for computer screens - min width 1025 ***/
+
+@media (min-width: 1025px) {
+    
+  .cards { 
+    grid-template-columns: repeat(4, 1fr);
+    grid-gap: 15px;
+    align-items: center;
+  }
+
+  .card { 
+    height: 35rem;
+    grid-column: span 1;
+    text-align: center;
+  }
+
+  p {
+    font-size: 20px;
+  }
+
+  header {  
+    display: flex;
+    margin: 20px;
+    height: 200px;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+   nav {
+      display: flex;
+      flex-direction: row;
+  }
+    
+  .photo-1 {
+      max-width: 700px;
+  }
+
+  .flex-parent {
+    flex-direction: row;
+    flex-wrap: nowrap;
+    align-items: center;
+  }
+
+  .flex-child {
+      height: 650px;
+      justify-content: center;
+  }
+    
+  .one {
+      width: 60%;
+  }
+
+  .two {
+      width: 40%;
+  }
+
+  .logo {
+     width: 90%;
+  } 
+    
+  .pauline {
+    width: 90%;
+  }  
+  
+  .course {
+    width: 90%;
+  }
+
+  .winner {
+    width: 90%;
+  }
+
+  video {
+    height: 80%;
+  }
+
+    /*** Styling for polaroid effect that only appears in computer width ***/  
+  }
+  
+  figure {
+    display: inline-block;
+    -webkit-box-shadow: 0 4px 6px rgba(0, 0, 0, .3);
+    -moz-box-shadow: 0 4px 6px rgba(0, 0, 0, .3);
+    box-shadow: 0 4px 6px rgba(0, 0, 0, .3);
+    -webkit-transition: all .20s linear;
+    -moz-transition: all .20s linear;
+    transition: all .20s linear;
+    cursor: pointer;
+    position: relative;
+  }
+
+  figcaption {
+    color: black;
+    font-size: 20px;
+    position: relative;
+    top: 15px;
+  }
```

Comment:
I can't see that you ever used the `<figcaption>`-tag in the HTML-file, did you perhaps change your mind and forget to remove this?